docs/guide/snaps.md

```diff
@@ -30,55 +31,38 @@ Among other things, SES allows us to restrict access to global JavaScript APIs a
 
 ## Getting Started
 
-There are many simple snaps available for you to learn from and begin to understand what your own snap needs. The following tutorial will walk you through a "Hello, World!" snap.
-
 ### Prerequisites
+- [Node.js 16.0](https://nodejs.org/) or later.
 
-- Node.js
-  - We recommend using [nvm](https://github.com/nvm-sh/nvm). If you don't want to use nvm, your local Node version needs to be greater than or equal to the one listed [here](https://github.com/MetaMask/template-snap/blob/main/.nvmrc).
-- git
-- [yarn v1](https://classic.yarnpkg.com)
-- Up-to-date Chromium or Firefox browser
-
-### Installing MetaMask Flask
-
-[MetaMask Flask](https://metamask.io/flask) is a separate extension for developers that provides access to additional unstable APIs. The goal of Flask is to maximize developer control, so that we can learn the full extent of what developers want to do with MetaMask and incorporate those lessons into the stable MetaMask distribution. Snaps is the first feature to be released on Flask.
-
-- First, in a Firefox or Chromium browser, create a new browser profile or disable any existing installed versions of MetaMask.
-  - Running multiple instances of MetaMask in the same browser profile will break dapp interactions.
-- Next, download [Flask](https://metamask.io/flask) and add it your chosen browser.
+### Install MetaMask Flask
 
-### Serving a Snap to Your Local Environment
+To interact with (your) Snaps, you will need to install [MetaMask Flask](https://metamask.io/flask/), a canary distribution for developers that provides access to upcoming features. 
 
-With Flask in your browser, it's time to plug something into it. If you don't want to download any code for the moment, check out the example snap [deployed here](https://metamask.github.io/template-snap/) and follow the instructions below under 'Interacting with the Snap'.
+Make sure to install Flask in a new browser profile or disable any existing installed versions of MetaMask. Running multiple instances of MetaMask in the same browser profile will break dapp interactions.
 
-If you're ready to jump in and see how a Snap looks under the hood, follow the steps below!
+### Quick start using our template
+Get started using our template built with TypeScript and React. Clone the repository via the command line or [via GitHub](https://github.com/MetaMask/template-snap-monorepo/generate).
+```shell
+git clone git@github.com:MetaMask/template-snap-monorepo.git    
+```
 
-Open up a new terminal window, and do the following:
 
-- Clone the [template snap](https://github.com/MetaMask/template-snap) repo.
```

Comment:
Instead of cloning the repo, should we just recommend they make a new repo using GitHub's template feature? Since we provide instructions to clone the repo on its own line, people might think that's the way we recommend. But if they really want to develop their own snap then they need to make their own repo, otherwise they'll try to push to our template and have to rejigger their Git configuration to work with their new repo.

So maybe this should just read:

``` suggestion
### Quick start using our template
[Create a new repo using our template](https://github.com/MetaMask/template-snap-monorepo/generate), then open up a new terminal window, and do the following:

- Clone your repo.
```

Reply:
Agreed, let's nudge users towards using the template feature.

Reply:
It's out of date but the gas-fee-snap tutorial has an example of showing users how to use the template: https://github.com/Montoya/gas-fee-snap#readme

One annoying thing about the GitHub template feature is that the button does not show up when you are logged out... that can be confusing for people. So best approach is to tell people both methods, or to explicitly tell them to log in to GitHub first.

Reply:
Gotcha, that's fine to list both methods then. That said I like the screenshot — it's so big that I think people will want to try that first rather than clone the repo.

Reply:
Good point. I'll change up the order so it's more of an alternative to using the template. I don't like the idea of asking people to log in in our docs.